README.md

```diff
@@ -1,185 +1,176 @@
 JavaScript Arrow Functions
 ---
 
-## Objectives
-
-1. Practice writing arrow functions
-2. Explain how arrow functions differ from named functions
-3. Describe situations where arrow functions come in handy
+## Problem Statement
 
-## Function, function, function, function, function
+You're familiar by now with the standard style of declaring functions in
+JavaScript.
 
-You're familiar by now with the standard `function foo() { return 'bar' }` style of functions in JavaScript.
-Well, there is another way to write functions in JavaScript called arrow functions:
-
-``` javascript
-// using our old standard function
-const oldStandardFunction = function() {
-  return "old standard functions rule!"
+```js
+function foo() {
+  return 'bar';
 }
-
-
-// updating to use an arrow function
-const arrowFunction = () => {
-  return 'Arrow functions are great!'
-};
 ```
 
-These are called [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) in reference to the little `=>` that characterizes them.
+But JavaScript has two other ways to write functions that are common in
+iteration and in writing event-handling code ("on click of this button, do
+that"). This lesson will teach those styles before you learn advanced iteration
```

Comment:
Unsure if we need the "before you learn" section here. It could be confusing and seems unnecessary to set up the topic at hand. It makes more sense when, for example, advanced iteration comes up later in the course of regular explanations.